Research/KitchenSinkQuantBookTemplate.ipynb

```diff
@@ -29,27 +29,69 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%matplotlib inline\n",
-    "# Imports\n",
-    "from clr import AddReference\n",
-    "AddReference(\"System\")\n",
-    "AddReference(\"QuantConnect.Common\")\n",
-    "AddReference(\"QuantConnect.Jupyter\")\n",
-    "AddReference(\"QuantConnect.Indicators\")\n",
-    "from System import *\n",
-    "from QuantConnect import *\n",
-    "from QuantConnect.Data.Custom import *\n",
-    "from QuantConnect.Data.Market import TradeBar, QuoteBar\n",
-    "from QuantConnect.Jupyter import *\n",
-    "from QuantConnect.Indicators import *\n",
-    "from datetime import datetime, timedelta\n",
-    "import matplotlib.pyplot as plt\n",
-    "import pandas as pd\n",
+    "# Move our working directory to where our Dll's are stored\n",
+    "# Only necessary when you are working in a different directory like this example\n",
+    "%cd ../Launcher/bin/Debug/"
```

Comment:
Not sure I would place this (same for C#), this isn't required for the docker images right? Maybe a comment in the next cell above `run/load` about having to adjust the path/location if not running in the docker image